lib/active_admin/views/components/paginated_collection.rb

```diff
@@ -76,6 +78,20 @@ def build_pagination_with_formats(options)
         end
       end
 
+      def build_per_page_select
+        div class: "pagination_per_page" do
+          text_node "Per page:"
+          select do
+            @per_page.each do |per_page|
+              option(per_page, {
+                value: per_page,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.